package/htmd/build.sh

```diff
@@ -39,6 +39,9 @@ else
     exit 1
 fi
 
+# Delete compiled libs from other OS
+find $DIR/htmd/lib -maxdepth 1 ! -name "$OSNAME" ! -path $DIR/htmd/lib -type d -exec rm -rf {} \; -print || true
```

Comment:
I don't get the `! -path $DIR/htmd/lib` part of the command

Reply:
find gives you always the source folder of search (`$DIR/htmd/lib` in this case). It's just a way of excluding it

Reply:
merci, didn't know that